sdk/src/openssl/temp_signer.rs

```diff
@@ -102,33 +96,20 @@ pub fn get_ec_signer<P: AsRef<Path>>(
     alg: &str,
     tsa_url: Option<String>,
 ) -> (EcSigner, PathBuf) {
-    let (key_name, ec_key_name) = match alg {
-        "es256" => ("ec256_key", "prime256v1"),
-        "es384" => ("ec384_key", "secp384r1"),
-        "es512" => ("ec512_key", "secp521r1"),
+    match alg {
+        "es256" | "es384" | "es512" => (),
         _ => {
             panic!("Unknown EC signer alg {:#?}", alg);
         }
-    };
```

Comment:
I may want to use this code in c2patool at some point, but I think not having it in the sdk is a good idea.